cadquery/cq.py

```diff
@@ -3432,43 +3455,45 @@ def sphere(
         :param angle3: The third angle to sweep the sphere arc through
         :type angle3: float > 0
         :param centered: A three-tuple of booleans that determines whether the sphere is centered
-            on each axis origin
+            on each axis origin, or a single bool
         :param combine: Whether the results should be combined with other solids on the stack
             (and each other)
         :type combine: true to combine shapes, false otherwise
+        :param clean: call :py:meth:`clean` afterwards to have a clean shape
         :return: A sphere object for each point on the stack
 
-        Centered is a tuple that describes whether the sphere should be centered on the x,y, and
-        z axes.  If true, the sphere is centered on the respective axis relative to the workplane
-        origin, if false, the workplane center will represent the lower bound of the resulting
-        sphere.
+        Centered is a tuple that describes whether the sphere should be centered on the x, y, and
+        z axes. If true, the sphere is centered on the respective axis relative to the workplane
+        origin. If false, the workplane center will represent the lower bound of the resulting
+        sphere. centered=True can be used as a shortcut for centered=(True, True, True) and
+        the same with False.
 
         One sphere is created for each item on the current stack. If no items are on the stack, one
         box using the current workplane center is created.
 
-        If combine is true, the result will be a single object on the stack:
-            If a solid was found in the chain, the result is that solid with all spheres produced
-            fused onto it otherwise, the result is the combination of all the produced boxes
+        If combine is true, the result will be a single object on the stack. If a solid was found
+        in the chain, the result is that solid with all spheres produced fused onto it otherwise,
+        the result is the combination of all the produced boxes.
```

Comment:
I think the "boxes" reference here may be a very old copy-paste error.

Reply:
Thanks, I must have read that 10+ times and it didn't click.